control-plane/agents/src/bin/ha/cluster/switchover.rs

```diff
@@ -237,8 +238,7 @@ impl SwitchOverRequest {
                                 info!("Retrying Republish after cleaning up targets");
                                 return Ok(());
                             }
-                            Err(e) => {
-                                error!("error while listing nvme controller {}", e.to_string());
+                            _ => {
```

Comment:
please also include error.to_string() or at least err.kind in the error we return below

Reply:
Done